src/test/resources/sample2vid.json

```diff
@@ -0,0 +1,114 @@
+{
+        "contigs": {
+                "22": {
+                        "length": 51304566,
+                        "tiledb_column_offset": 0
+                }
+        },
+        "callset_mapping_file": "src/test/resources/sample2callsets.json",
+        "fields": {
+                "AD": {
+                        "vcf_field_class": ["FORMAT"],
+                        "type": "int"
+                },
+                "DP": {
+                        "vcf_field_class": ["FORMAT", "INFO"],
+                        "type": "int",
+			"VCF_field_combine_operation": "sum"
+                },
+                "GQ": {
+                        "vcf_field_class": ["FORMAT"],
+                        "type": "int"
+                },
+                "GT": {
+			"vcf_field_class": ["FORMAT"],
+                        "type": "int",
+                        "length": "P"
+                },
+                "PL": {
+                        "vcf_field_class": ["FORMAT"],
+                        "type": "int",
+                        "length": "G"
+                },
+		"AC": {
+                        "vcf_field_class": ["INFO"],
+                        "type": "int",
+			"length": "A",
+			"VCF_field_combine_operation": "median"
```

Comment:
I guess I'm confused why `AC` is `median`. The AC is only on the variant's info field (right?), so what values are we taking the median of? I realize this might not have a sensible answer because genomics db is weird, but I want to make sure I understand the source of the weirdness.